main.go

```diff
@@ -266,7 +275,7 @@ func renderHandler(w http.ResponseWriter, r *http.Request) {
 	if tstr := r.FormValue("cacheTimeout"); tstr != "" {
 		t, err := strconv.Atoi(tstr)
 		if err != nil {
-			logger.Error("failed to parse cacheTimeout",
+			logger.Error("Failed to parse cacheTimeout",
```

Comment:
I'd say that it's better to have start all strings with a small letter. E.x. 'failed to parse' instead of "Failed to parse".
That's also about "Reason" field.